### PR TITLE
Addresses Issue #741 - [elbow visualizer] Error in distortion score calculation

### PR DESCRIPTION
**Pull request by pdamodaran**
As reported by @maximilian1064 and confirmed by @zjpoh, the current version of the elbow visualizer was incorrectly calculating the distortion score.  

The line distortion += distances.mean() needed to be replaced with distortion += distances.sum()

This pull request makes the fix and it also updates the expected results of the tests in test_elbow.py based on the fix made to elbow.py.

### CODE REVIEW

File: yellowbrick/cluster/elbow.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def distortion_score(X, labels, metric='euclidean'):
         distances = distances ** 2
 
         # Add the mean square distance to the distortion
```
Comment (bbengfort) [inline yellowbrick/cluster/elbow.py]: Thank you for making this change - would you mind also updating the comment above to reflect the new sum? I know it's small - but I just want to make sure we're consistent!

File: tests/test_cluster/test_elbow.py [hunk 2]
```diff
@@ -221,7 +221,7 @@ def test_distortion_metric(self):
         )
         visualizer.fit(X)
 
-        expected = np.array([ 7.677785,  8.364319,  8.893634,  8.013021])
+        expected = np.array([ 69.100065, 54.081571, 43.146921, 34.978487])
```
Comment (bbengfort) [inline tests/test_cluster/test_elbow.py]: Thank you for updating all these values!

File: yellowbrick/features/pca.py [hunk 3]
```diff
@@ -24,6 +24,7 @@
 from yellowbrick.features.base import MultiFeatureVisualizer
 from yellowbrick.style import palettes
 from yellowbrick.exceptions import YellowbrickValueError
+from yellowbrick.draw import manual_legend
```
Comment (bbengfort) [inline yellowbrick/features/pca.py]: This file looks like work from a different PR/feature? Would you mind reverting it so we don't include these changes in this PR?

Comment (pdamodaran) [reply yellowbrick/features/pca.py]: @bbengfort - I copied the latest code but it looks like there is a hidden character in the last line.  I am not sure how to fix it.  Do you have any ideas?

Comment (bbengfort) [reply yellowbrick/features/pca.py]: Add a newline to the end of the file? I think that should fix it.

**Pull request closed without merging**